spec/fixtures/meow.json

```diff
@@ -1 +1,17 @@
+/* 
```

Comment:
I don't think we really need the headers in the fixtures.

Reply:
:+1: It looks like three more json fixtures could have headers removed: structure.json, structure_with_html.json, and waveform.json.